### PR TITLE
ONI-107: HF status picker.

### PR DESCRIPTION
**Pull request by wzglinieckisoldevelo**
Ticket: https://openimis.atlassian.net/browse/ONI-107

Dependencies: https://github.com/openimis/openimis-be-location_py/pull/86

Changes:
- Added HF status picker 
- Relevant settings, filters, constants added.

### CODE REVIEW

File: src/components/HealthFacilityFilter.js [hunk 1]
```diff
@@ -246,6 +247,16 @@ class HealthFacilityFilter extends Component {
             }
           />
         </Grid>
+        {this.isHealthFacilityStatusEnabled && <Grid item xs={3} className={classes.item}>
+          <PublishedComponent
+            module="location"
+            label="HealthFacilityForm.status"
+            pubRef="location.HealthFacilityStatusPicker"
+            value={this._filterValue("status")}
+            onChange={(v, s) => this._onChange("status", v, s)}
```
Comment (olewandowski1) [inline src/components/HealthFacilityFilter.js:256]: Use more explicit name for 'v'. I believe 's' argument is not necessary here.

File: src/pickers/HealthFacilityStatusPicker.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+        module="location"
+        label="healthFacilityStatus"
+        constants={ HEALTH_FACILITY_STATUSES }
+        {...this.props}
+      />
+    );
+  }
+}
+
+export default HealthFacilityStatusPicker;
```
Comment (olewandowski1) [inline src/pickers/HealthFacilityStatusPicker.js:19]: Change it to function component.

File: src/components/HealthFacilityFilter.js (hunk 1 above)
Comment (wzglinieckisoldevelo) [reply src/components/HealthFacilityFilter.js]: I would like to omit the "s" part here, I would have to do a little refactor to the whole component. I want to keep things separate between PR's.

**Review by olewandowski1 — APPROVED** [review]
LGTM @wzglinieckisoldevelo

**Pull request merged**